### PR TITLE
[CHI-133] Add anchor links to documentation heading tags

### PR DESCRIPTION
**Pull request by davidojedacl**

### CODE REVIEW

**Comment by davidojedacl** [comment]
@mattnickles @jllr I have reopened the PR to solve the problem with copy url link to acces directly

File: src/website/assets/scripts/chi-docs.js [hunk 1]
```diff
@@ -108,6 +112,29 @@ onLoad(() => {
     };
   });
 
+  var anchors = document.querySelectorAll('h3,h4');
```
Comment (mattnickles) [inline src/website/assets/scripts/chi-docs.js]: The heading tag example on the typography page is not getting excluded from the anchor function. 
image

File: src/website/assets/styles/chi-docs.scss [hunk 2]
```diff
@@ -79,4 +79,11 @@ body {
   .a-table__options code {
     word-break: normal;
   }
+
+  .-anchor {
+    &::before {
+      height: 5rem;
```
Comment (jllr) [inline src/website/assets/styles/chi-docs.scss]: @davidojedacl this should work from lg-breakpoint (992px), not before this point

**Comment by jllr** [comment]
@davidojedacl found a bug when there's a match in anchors in the same page (i.e Typography > Base anchor)

File: src/website/assets/styles/chi-docs.scss (hunk 2 above)
Comment (davidojedacl) [reply src/website/assets/styles/chi-docs.scss]: done!

File: src/website/assets/scripts/chi-docs.js (hunk 1 above)
Comment (davidojedacl) [reply src/website/assets/scripts/chi-docs.js]: done!

**Pull request merged**